config/initializers/timezone.rb

```diff
@@ -0,0 +1,2 @@
+# Set the timezone for the JavascriptAgent (libv8 only relies on the TZ variable)
+ENV['TZ'] = Time.zone.tzinfo.canonical_identifier
```

Comment:
Not sure, shouldn't the `TIMEZONE` `.env` setting be the only source for the timezone Huginn uses?